healthcheck.go

```diff
@@ -1,118 +1,92 @@
 package main
 
 import (
-	"encoding/json"
-	"errors"
-	"fmt"
-	health "github.com/Financial-Times/go-fthealth/v1_1"
+	"time"
+	"net/http"
+
+	fthealth "github.com/Financial-Times/go-fthealth/v1_1"
 	"github.com/Financial-Times/message-queue-gonsumer/consumer"
+	"github.com/Financial-Times/message-queue-go-producer/producer"
 	"github.com/Financial-Times/service-status-go/gtg"
-	"github.com/Sirupsen/logrus"
-	"io/ioutil"
-	"net/http"
 )
 
-const healthPath = "/__health"
-
-type healthService struct {
-	httpClient     *http.Client
-	consumerConfig consumer.QueueConfig
-	config         *healthConfig
-	checks         []health.Check
-}
+const (
+	healthPath = "/__health"
+    requestTimeout = 4500
+)
 
-type healthConfig struct {
+type HealthCheck struct {
+	consumer consumer.MessageConsumer
+	producer producer.MessageProducer
 	appSystemCode string
 	appName       string
 }
 
-func newHealthService(config *healthConfig, httpClient *http.Client, consumerConfig consumer.QueueConfig) *healthService {
-	service := &healthService{
-		config:         config,
-		httpClient:     httpClient,
-		consumerConfig: consumerConfig,
+func NewHealthCheck(producerConf *producer.MessageProducerConfig, consumerConf *consumer.QueueConfig, appSystemCode, appName string) *HealthCheck {
```

Comment:
a `MessageProducer` and `MessageConsumer` are available in `main.go`. Pass those to the method instead of the `MessageProducerConfig` and `QueueConfig` so that:
- the actual producer and consumer used by the app are also used by the healtchecks
- we simplify the code
- we don create an extra httpClient only for the healthchecks